tests/unittests/BackendTestUtils2.h

```diff
@@ -298,6 +296,12 @@ void inferExtract3D(Tensor *input, Tensor *out, llvm::StringRef kind);
 
 void inferMaxSplat(Tensor *input, Tensor *out, llvm::StringRef kind);
 
+void insertCompiledFunction(llvm::StringRef name, CompiledFunction *func,
```

Comment:
Looks like this is unused. Is it going to be used by a future PR, if so maybe add it there?